frontend/components/badge-navigation/component.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Link from 'next/link';
+
+import Tag from 'components/tag';
+
+import { BadgeNavigationProps } from './types';
+
+export const BadgeNavigation: FC<BadgeNavigationProps> = ({
+  className,
+  activeId,
+  items,
+}: BadgeNavigationProps) => {
+  const intl = useIntl();
+
+  return (
+    <div className={className}>
+      <nav aria-label={intl.formatMessage({ defaultMessage: 'Navigation', id: 'fBg+7V' })}>
```

Comment:
I think this `aria-label` is redundant. If not mistaken, the screen reader will already announce the `<nav />` element as a “navigation”.